tests/components/rfxtrx/__init__.py

```diff
@@ -6,7 +6,7 @@ async def _signal_event(hass, packet_id):
     event = rfxtrx.get_rfx_object(packet_id)
 
     await hass.async_add_executor_job(
-        hass.data[rfxtrx.DATA_RFXOBJECT].event_callback, event,
+        hass.data[rfxtrx.DOMAIN][rfxtrx.DATA_RFXOBJECT].event_callback, event,
```

Comment:
It would be good to clean this up soon so we don't need to change this when changing the integration.

https://github.com/home-assistant/core/pull/37565#discussion_r453380566

Reply:
Working on it.

Reply:
https://github.com/home-assistant/core/pull/38041